libraries/AP_RCProtocol/AP_RCProtocol_FPort.cpp

```diff
@@ -37,8 +38,9 @@ extern const AP_HAL::HAL& hal;
 #define FLAGS_FAILSAFE_BIT	3
 #define FLAGS_FRAMELOST_BIT	2
 
-#define CHAN_SCALE_FACTOR ((2000.0 - 1000.0) / (1800.0 - 200.0))
```

Comment:
I'm guessing someone put this derivation in for a reason at some stage.  If they'd left a comment that would have helped.

Reply:
it's a fixed property of the protocol, and I think the constant is just as clear